projects/storefrontlib/shared/components/carousel/carousel.component.html

```diff
@@ -104,31 +104,61 @@ <h2 *ngIf="title">{{ title }}</h2>
     <ng-container *ngFor="let _ of items; let i = index">
       <!-- TODO: (CXSPA-6904) - Remove feature flag next major release -->
       <ng-container *cxFeature="'a11yFocusableCarouselControls'">
-        <button
-          *ngIf="i % size === 0"
-          (click)="$event.stopPropagation(); activeSlide = i"
-          [attr.aria-selected]="i === activeSlide"
-          [attr.aria-disabled]="i === activeSlide"
-          [attr.aria-label]="
-            'carousel.slideNumber'
-              | cxTranslate
-                : {
-                    currentSlideNumber: getSlideNumber(size, i),
-                    numberOfSlides: size,
-                  }
-          "
-          [attr.title]="
-            'carousel.slideNumber'
-              | cxTranslate
-                : {
-                    currentSlideNumber: getSlideNumber(size, i),
-                    numberOfSlides: size,
-                  }
-          "
-          role="tab"
```

Comment:
I believe this has been included to mimic the [tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) as we decided this was best at the time. If we are moving away from it we probably should also remove `role="tablist"` from the wrapper div. 

I think removing those roles isn't a breaking change. We can get away with not using a feature flag here at least we did so in the past 😉 .